lib/expand.js

```diff
@@ -143,12 +143,31 @@ api.expand = ({
       {activeCtx, localCtx: element['@context'], options});
   }
 
+  let keys = Object.keys(element).sort();
+
+  // look for scoped context on @type
+  for(let ki = 0; ki < keys.length; ++ki) {
```

Comment:
If you don't need the `ki` index var, use `for...of` and declare your `const key` within it:

```js
for(const key of keys) {
   // ...
```